configs/freebind/freebind.yaml

```diff
@@ -0,0 +1,39 @@
+admin:
```

Comment:
Quick drive by: Can we test this config (just that it loads) in our config tests? Along with the original_dst config? From a quick look it doesn't seem like they are being tested for config sanity.

Reply:
OK SGTM.

Reply:
+1 - let's avoid text config because when it stops working we won't be alerted but a TODO is fine.

I think we do need to be able to test socket options, esp with all the other PRs in flight